mteb/tasks/Classification/eng/LegalBenchClassification.py

```diff
@@ -1446,8 +1446,8 @@ class CUADExpirationDateLegalBenchClassification(AbsTaskClassification):
             year={2021}
         }
         """,
-        n_samples={"test": 1216},
-        avg_character_length={"test": 337.14},
+        n_samples={"test": 876},
+        avg_character_length={"test": 309.27},
```

Comment:
How come these change?

Reply:
Were they just incorrect beforehand?

Reply:
Yes, I forgot to update them when I pushed them. I recomputed the values for each, and now they reflect the accurate values.